config/database.yml

```diff
@@ -58,6 +58,11 @@ development:
 test:
   <<: *default
   database: vglist_test
+  username: postgres
+  
+  host: <%= ENV['VGLIST_DATABASE_HOST'] || 'localhost' %>
```

Comment:
IDK if it's easier to just include this and the password in the default block at this point 🤔